esmvaltool/preprocessor/_regrid.py

```diff
@@ -15,11 +15,11 @@
 
 import iris
 import iris.exceptions
+from iris.analysis import AreaWeighted, Linear, Nearest, UnstructuredNearest
```

Comment:
Please sort the imports using `isort`

Reply:
as you wish (as in done!) but my prospector is complaining the imports are not grouped :cry:

Reply:
`isort` is more right than `prospector`